index.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Delicious</title>
+        <meta name="description" content="Dessert Recipe">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="shortcut icon" href="favicon.ico" type="image/png">
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <header>
+            <a href="#"><img src="logo.png" alt="fork and knife logo"></a>
+            <h1>Delicious</h1>
+            <p>The best food blog on the web</p>
+            <!-- Row of share icons floated to the right -->
+            <nav>
+                <a href="http://facebook.com"><img src="img/fb-icon.png" alt="Facebook"></a>
```

Comment:
The design pattern for these 'nav' elements involves a ul/li for layout, unwieldy as it seems. So:
```
<nav>
  <ul>
    <li><a href=""><img src="" alt=""></a></li>
  </ui>
</nav>
```
It is... sort of weird, but has some benefits later on for JS/jQ that we'll hopefully get into.